src/test/clojure/clara/test_durability.clj

```diff
@@ -336,12 +339,13 @@
 (deftest test-assemble-restored-session-opts
   (let [orig (mk-session 'clara.durability-rules)
 
-        test-assemble (fn [rulebase memory]
-                        (let [activation-group-fn-called? (volatile! false)
+        test-assemble (fn [orig assemble-with-memory?]
+                        (let [memory (.memory orig)
```

Comment:
I'd prefer to avoid the interop/reflection and instead go with 
`(-> orig eng/components :memory)`

It is more the public API anyways.